erpnext/stock/doctype/purchase_receipt/purchase_receipt.py

```diff
@@ -467,25 +465,11 @@ def update_billing_status(self, update_modified=True):
 
 		self.load_from_db()
 
-	def create_package_tag(self):
+	def validate_duplicate_package_tags(self):
 		package_tags = [item.package_tag for item in self.items if item.package_tag]
-
 		if len(package_tags) != len(set(package_tags)):
 			duplicate_tags = list(set([tag for tag in package_tags if package_tags.count(tag) > 1]))
-			frappe.throw("Package Tag {0} cannot be same for multiple Purchase Receipt Item".format(", ".join(duplicate_tags)))
-
-		for item in self.items:
-			if item.package_tag:
-				package_tag = frappe.db.exists("Package Tag", {"package_tag": item.package_tag})
-				if package_tag:
-					frappe.throw("Row #{0}: Package Tag '{1}' already exists".format(item.idx, frappe.utils.get_link_to_form("Package Tag", package_tag)))
-				else:
-					doc = frappe.new_doc("Package Tag")
-					doc.update({
-						"package_tag": item.package_tag,
-						"item_code": item.item_code
```

Comment:
You will have to add logic to ensure that the package tag is linked with the item and batch when the Purchase Receipt is submitted